integrations/tensorflow/test/iree_tfl_tests/cartoon_gan.run

```diff
@@ -1 +1,2 @@
 # RUN: %PYTHON -m iree_tfl_tests.cartoon_gan_test --artifacts_dir=%t
+# XFAIL: *
```

Comment:
If we have an issue for this failure, we may want to add a `# TODO(issue#): ...` on top of this so we will remember to remove xfail.

Reply:
Ok good point.. Let me see if there are any failures. I'll do it after the fact.